guides/delivery/setting-up-a-new-project.md

```diff
@@ -0,0 +1,88 @@
+# Setting up a new project 
+
+At dxw we frequently start new projects and work with clients we haven’t worked with before. 
+
+Delivery managers at dxw are responsible for setting up projects and ensuring all the right things are in place for when the team starts work. 
+
+We often find that time is limited and transitioning between projects can be a challenge, for not only delivery managers but also the wider team.
+
+This guide provides advice and tips, as well as a checklist for delivery managers about to start a new project. It can work for large projects (a phase of work like a discovery or beta) as well as smaller projects which might only be a sprint or two.
+
+## Before a project starts
+
+In most cases, there’s not a very long lead time between winning a piece of work and starting it. We sometimes have a number of days, rather than weeks, for the team to get everything in place. This includes things like contracts, scheduling a team to do the work and making a plan for the first few days. 
+
+We’re lucky if the same delivery manager has been involved in the sales work that led to the project being won, because there’s always other work in flight. It’s important that knowledge is shared and the necessary introductions made so we can get the work off to a good start.
+
+It is however worth remembering that most of our sales proposals are produced with limited information about the project, and it’s only once we start that we get a better idea of what the work is and what skills we’ll need to do it. 
+
+There are several things that delivery managers should do first, regardless of the shape of the project:
+
+### Do some reading 
+
+Get hold of any background materials, such as proposal documents, meeting notes, project documents or email chains. Familiarise yourself with the brief and conversations that have taken place so far
```

Comment:
For consistency, would you mind adding a period at the end of this paragraph?

Reply:
Thanks, done!